plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -370,6 +370,50 @@ fu_plugin_uefi_capsule_update_splash (FuPlugin *plugin, FuDevice *device, GError
 	return fu_plugin_uefi_capsule_write_splash_data (plugin, device, image_bmp, error);
 }
 
+static gboolean
+fu_capsule_update_on_disk (FuDevice *device, GError **error)
```

Comment:
I think this function would be better to have an out pointer for the actual result, and then save the TRUE/FALSE for success/failure in setting it

Reply:
sure